### ISSUE TITLE
debugging warnings in 4.2

### ISSUE THREAD

**Issue by danmarsden** [ethz-let/quiz-report_filedownloader#2]
This file is no longer required in Moodle 4.2+. Please do not include/require it.
line 25 of /mod/quiz/report/attemptsreport.php: call to debugging()
line 28 of /mod/quiz/report/filedownloader/report.php: call to require_once()
line 88 of /mod/quiz/report.php: call to include_once()

Class 'quiz_attempts_report' has been renamed for the autoloader and is now deprecated. Please use 'mod_quiz\local\reports\attempts_report' instead.
line 145 of /lib/classes/component.php: call to debugging()
line ? of unknownfile: call to core_component::classloader()
line 34 of /mod/quiz/report/filedownloader/report.php: call to spl_autoload_call()
line 88 of /mod/quiz/report.php: call to include_once()

not a blocker for plugins db approval. - just an FYI

**Comment by nexterday** [comment]
Yes Dan, we noticed that last week, and we are noting all other plugins we are having for un update. We are still currently run on 3.11, hence the process of those updates takes slightly longer route.

I'll leave the issue open until we address it without affecting current moodle 3.x users. 

Thanks again!
